### PR TITLE
Add/refactor 01-Visualization chapter

### PR DESCRIPTION
**Pull request by pat-s**
- adds content for day1 10-12am

- adds a `.bib` file for references

I merged the already existing content with new content. 
The only file I removed completely was `14-stat` since this content now lives within `12-geoms`.

### CODE REVIEW

**Comment by krlmlr** [comment]
Thanks. At which stage do you intend to cover faceting, if at all?

**Comment by pat-s** [comment]
Scheduled for day 2 10:50-12:20.
My plan was to add it in "Chapter 5 - visualization advanced" to stay in line with the time schedule.

Otherwise append it to the "chapter 2 visualiuation basics" and rename.

Will add content tmrw.

**Comment by krlmlr** [comment]
Thanks!

I think this is such a low hanging fruit that it absolutely must come on day one, but we can discuss.

File: book/script/13-export.Rmd [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+- saving in pixels (differs on every monitors)
+
+- saving as a square image
+
+- no option to specify the DPI (dots per inch)
+
+### Storing the plot as an R object
+
+One of the major advantages of \colorbox{github}{\texttt{ggplot()}} is that you can save a plot as an R object and modify it later.
```
Comment (krlmlr) [inline book/script/13-export.Rmd:58]: This needs to work on both HTML and PDF.

Comment (pat-s) [reply book/script/13-export.Rmd:58]: This is a leftover. If you refer to `\colorbox{github}{\texttt{ggplot()}}`.

**Comment by pat-s** [comment]
> I think this is such a low hanging fruit that it absolutely must come on day one, but we can discuss.

Facetting? AFAIR we agreed on having an "advanced ggplot2" part on day2 including facetting and extensions?

File: book/script/13-export.Rmd (hunk 1 above)
Comment (pat-s) [reply book/script/13-export.Rmd:58]: Ah, you fixed it already in master. :+1:

**Pull request merged**